infrastructure/ansible/group_vars/production.yml

```diff
@@ -1,7 +1,2 @@
-# TODO: get these values from environment variables
-papertrail_token: ""
-lobby_postgres_user: ""
-lobby_postgres_password: ""
-lobby_postgres_port: ""
-http_server_error_report_github_token: ""
 http_server_error_report_github_repo: "triplea"
+bot_lobby_address: production.triplea-game.org
```

Comment:
This DNS entry doesn't exist (yet?)
Do you plan on adding it?

Reply:
Good point/catch. The production configuration is not yet fully baked, we'll need to spend some time to ensure it is right when we get closer to deployment time. I'm thinking/planning for us to deploy to different servers to help make sure we do not have any kinks/problems such as a non-existent DNS.

With that said, it's a bit of an open question of what the DNS name should be for prod. I'm wondering if we should bake in the lobby version number to the DNS name so we can distinguish between the multiple production versions, or if even that makes sense. We could use 'project' names to refer to the different production versions of TripleA.

Reply:
I mean if we're really crazy we could also use the namecheap api to have the scripts auto-manage dns entries for us:
https://www.namecheap.com/support/api/methods/domains-dns/set-hosts.aspx

This way baking in the number wouldn't be that big of a deal for the system.

For the user that's another problem though